src/extension/index.ts

```diff
@@ -48,6 +52,65 @@ export async function activate(context: ExtensionContext) {
     const panel = new Panel(context, baser, store);
     disposables.push(commands.registerCommand('sarif.showPanel', () => panel.show()));
 
+    // URI handler
+    window.registerUriHandler({
+        async handleUri(uri: vscode.Uri) {
+            if (uri.path === '/alert') {
```

Comment:
This will handle all URIs with `/alert` as its path, I think. So, `http://evil.com/alert` would be accepted. Is there a way to be more specific?

Reply:
This will be the vscode:// uri, in this case vscode://MS-SarifVSCode.sarif-viewer/alert?url=...

Reply:
hey I found below see if it fits:

```
if (uri.scheme === 'myextension') {
        if (uri.path === '/open-file') {
    }
}
```